apps/nrm/views/current-pv-location.html

```diff
@@ -1,6 +1,6 @@
 {{<partials-page}}
   {{$page-content}}
-  <p>{{#markdown}}current-pv-location{{/markdown}}</p>
+  <p>{{#t}}pages.current-pv-location.heading-1{{/t}}</p>
```

Comment:
markdown is fine and actually better for long text https://ukhomeofficeforms.github.io/hof-guide/documentation/#free-text-content